library/asb_get_state.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+import os
+
+from ansible.module_utils.basic import AnsibleModule
+try:
+    from kubernetes import client, config
+    from kubernetes.client.rest import ApiException
+    config.load_kube_config()
+    api = client.CoreV1Api()
+except Exception as error:
+    ansible_module.fail_json(
+        msg="Error attempting to load kubernetes client: {}".format(error))
+
+
+ENV_NAME = "POD_NAME"
+ENV_NAMESPACE = "POD_NAMESPACE"
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            key=dict(required=True, type='str'),
```

Comment:
wonder if we should make this accept a list?

Reply:
Now this might make more sense. Say they provide a list of keys which they would like to fetch and this is stored in a single return value?